src/main/java/com/minecolonies/coremod/colony/WorkManager.java

```diff
@@ -1,7 +1,7 @@
 package com.minecolonies.coremod.colony;
 
+import com.minecolonies.api.util.Log;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.util.Log'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)